xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/JoinWiki.xml

```diff
@@ -105,17 +105,12 @@
         ## An invitation exists.
         #if (($request.acceptInvitation || $request.rejectInvitation) &amp;&amp; $services.csrf.isTokenValid($form_token))
           ##
-          ## The user decided whether to accept or to reject the invitation.
-          #set ($notificationMessage = $services.localization.render('platform.wiki.users.invitation.userAcceptedNotification', [${xwiki.getUserName($currentUser, false)}, $currentUser, ${wikiName}, $wikiId]))
           ## Accept
           #if ($request.acceptInvitation)
             #set ($discard = $services.wiki.user.acceptInvitation($existingInvitation, $request.userComment))
           #else
             #set ($discard = $services.wiki.user.refuseInvitation($existingInvitation, $request.userComment))
-          #end          
-          ## Send message to wiki admins. (XWikiAdminGroup)
-          #set ($wikiAdminGroupReference = $services.model.createDocumentReference($wikiId, 'XWiki', 'XWikiAdminGroup'))
-          #set ($discard = $services.messageStream.postMessageToGroup($notificationMessage, $wikiAdminGroupReference))
+          #end
```

Comment:
Isn't removing this code a regression? Or is this somehow added back in the legacy module?

Reply:
See https://jira.xwiki.org/browse/XWIKI-22478

Reply:
and also: https://forum.xwiki.org/t/deprecate-and-remove-message-stream-from-xwiki-standard/14780/9?u=surli

TL;DR: the feature is broken for a long time now so removing it isn't causing a regression and we need a proper improvment for it.